packages/connector/server/src/services/ProfileService.kt

```diff
@@ -90,15 +90,16 @@ class ProfileService(
     override suspend fun upsertIntegration(request: UpsertIntegrationRequest): Integration {
         return withErrorWrapper {
             val f = request.descriptorForType.findFieldByName("id")
-            val integration = IntegrationEntity(
+            var integration = IntegrationEntity(
                 id = if (request.hasField(f)) request.id.value else null,
                 profileId = config.workspaceId,
                 type = request.type,
                 configs = request.configsList.associate { it.key to it.value }
             )
+            integration = profileManager.upsertIntegration(integration)
             // TODO: load the integration matched the type
             dbtManager.reloadContext()
-            buildProtoIntegration(profileManager.upsertIntegration(integration))
+            buildProtoIntegration(integration)
```

Comment:
This is a bug, the context of dbt can't reload if it executes before upserting integration.

Reply:
maybe this bugfix can be introduced by another Pull Request?
bugfix should be merged into main and be ready for the release tmr.
This PR should be merged in 0.8.2 maybe?